brambling/views/core.py

```diff
@@ -57,6 +58,10 @@ def get_context_data(self):
                 order.event = re_dict[order.event_id]
                 order.event.order = order
 
+            # Exclude registered events
+            upcoming_events_interest = upcoming_events_interest.exclude(pk__in=registered_events_qs)
+            upcoming_events = upcoming_events.exclude(pk__in=registered_events_qs)
```

Comment:
the list version... should work? If it doesn't it'll be real obvious real fast. we'd really need to do tests to know for sure which is faster, and at our current scale it probably doesn't matter.